actions.json

```diff
@@ -0,0 +1,47 @@
+[{
+        "id": "update_turnserver_ip",
+        "name": "Update turnserver ip",
+        "command": "/opt/yunohost/matrix-$YNH_APP_INSTANCE_NAME/Coturn_config_rotate.sh",
+        "user": "root",
+        "accepted_return_codes": [0],
+        "description": {
+            "en": "Update the ip in the turnserver config"
+        }
+},{
+        "id": "open_turnserver_firewall_ports",
+        "name": "Open ports for turnserver",
+        "command": "yunohost firewall allow Both 49153:49193",
+        "user": "root",
+        "accepted_return_codes": [0],
+        "description": {
+            "en": "Open the ports range 49153:49193 with TCP and UDP to be able to use correctly the turnserver."
+        }
+},{
+        "id": "close_turnserver_firewall_ports",
+        "name": "Close ports for turnserver",
+        "command": "yunohost firewall disallow Both 49153:49193",
+        "user": "root",
+        "accepted_return_codes": [0],
+        "description": {
+            "en": "Close the ports range 49153:49193 with TCP and UDP. (Undo \"Open ports for turnserver\" action)"
+        }
+},{
+        "id": "set_admin_user",
+        "name": "Set a user as admin",
+        "command": "[[ \"$(su --command=\"psql matrix_synapse\" postgres <<< \"UPDATE users SET admin = 1 WHERE name = '@$YNH_ACTION_USERNAME:$(yunohost app setting $YNH_APP_INSTANCE_NAME special_domain)'\")\" == 'UPDATE 1' ]]",
```

Comment:
It's not really normalised yet, but why you don't create a file for each actions, and call this one here instead of a "long" command like that?

I think about something like that:
https://github.com/YunoHost-Apps/leed_ynh/blob/9daea0a8492eaee4e781b8ece458fb628df5d8af/actions.json#L4

Reply:
You still can do an other PR to improve that if you want.